regress/gftool/gfuser/gfuser-A.auth_type.wrong.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+. ./regress.conf
+
+user=tmpusr-`hostname`-$$
+real=tmprealname
+home=/home/$user
+gsi_dn=""
+auth_id="auth_id"
+
+if $regress/bin/am_I_gfarmadm; then
+    :
+else
+    exit $exit_unsupported
+fi
+
+trap 'gfuser -d "$user" 2>/dev/null; rm -f $localtmp; exit $exit_trap' $trap_sigs
```

Comment:
Please add a newline like the following, to make checkpatch.pl happy:

	trap 'gfuser -d "$user" 2>/dev/null; rm -f $localtmp;
	    exit $exit_trap' $trap_sigs

Reply:
Please add a newline like the following, to make checkpatch.pl happy:

	trap 'gfuser -d "$user" 2>/dev/null; rm -f $localtmp;
	    exit $exit_trap' $trap_sigs